common/src/main/java/com/wynntils/features/inventory/CustomBankPagesFeature.java

```diff
@@ -323,16 +170,8 @@ private List<Integer> parseStringToDestinations(String destinationsStr, Searchab
 
             if (destinations.size() < MAX_DESTINATIONS) {
                 int startIndex = destinations.size();
-                List<Integer> defaultValues;
-
-                switch (containerType) {
-                    case BANK -> defaultValues = QUICK_JUMP_DESTINATIONS;
-                    case BLOCK_BANK -> defaultValues = BLOCK_BANK_DESTINATIONS;
-                    case BOOKSHELF, MISC_BUCKET -> defaultValues = HOUSING_DEFAULT_DESTINATIONS;
-                    default -> {
-                        return null;
-                    }
-                }
+                List<Integer> defaultValues =
+                        Models.ContainerQuickJump.getJumpDestinations(Models.Container.getContainerType());
```

Comment:
This will error as whilst in the config screen we aren't in one of the containers

Reply:
thx for reviewing this